index.js

```diff
@@ -26,12 +32,13 @@ module.exports = function(app) {
             "key": {
               title: "Path",
               type: "string",
-              default: ""
+              default: "",
             },
             "zones": {
               "type": "array",
-              "title": " ",
-              "description": "Zones",
+              "minItems": 1,
```

Comment:
Enabled by default and minItems > 0 are technically contradictory, as the plugin will be enabled but with empty configuration that does not allow you to hit `Submit` in plugin config form. Not a big deal, but may be a bit confusing. This also does not allow saving a configuration where you've deleted all zones for some reason or another.

If the user does not understand what they are doing this may cause more confusion than helping them. What do you think?

Reply:
To me that appeared as the best compromise. If you use the plugin, you need at least one path zones and one zone. Else disable the plugin. That said, the plugin starts fine with no path.

True that Min 1 will not allow you to submit, or go from 1 to 0, but there is a validation message in the UI that helps. We could suggest to deactivate the plugin in the error message instead of deleting the last path zones.

But I'm scared to even try to attempt to tweak form error messages! It's not well documented online. If you know how, your help is welcome.